crash with syslog tls

crash on syslog server refused

```
ERROR: 2024/01/19 20:38:05.405053 [syslog] logger=syslog - write error dial tcp 127.0.0.1:6514: connect: connection refused
INFO: 2024/01/19 20:38:05.405123 [syslog] logger=syslog - connecting to tcp+tls://127.0.0.1:6514 ...
ERROR: 2024/01/19 20:38:05.405160 [syslog] logger=syslog - write error dial tcp 127.0.0.1:6514: connect: connection refused
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x88 pc=0x104c257]

goroutine 29 [running]:
github.com/dmachard/go-clientsyslog.(*Writer).getConn(0xc000176cd0?)
        /home/denis/go/pkg/mod/github.com/dmachard/go-clientsyslog@v0.3.0/writer.go:32 +0x17
github.com/dmachard/go-clientsyslog.(*Writer).Close(0xc0000e2720?)
        /home/denis/go/pkg/mod/github.com/dmachard/go-clientsyslog@v0.3.0/writer.go:101 +0x18
github.com/dmachard/go-dnscollector/loggers.(*Syslog).FlushBuffer(0xc0001a6000, 0xc0001efac0)
        /home/denis/Lab/github.com/go-dns-collector/loggers/syslog.go:374 +0x51f
github.com/dmachard/go-dnscollector/loggers.(*Syslog).Process(0xc0001a6000)
        /home/denis/Lab/github.com/go-dns-collector/loggers/syslog.go:435 +0x39f
created by github.com/dmachard/go-dnscollector/loggers.(*Syslog).Run in goroutine 24
        /home/denis/Lab/github.com/go-dns-collector/loggers/syslog.go:272 +0x279
exit status 2
```

--- a/loggers/syslog.go
+++ b/loggers/syslog.go
@@ -371,8 +371,8 @@ func (s *Syslog) FlushBuffer(buf *[]dnsutils.DNSMessage) {
 		if err != nil {
 			s.LogError("write error %s", err)
 			s.syslogReady = false
-			s.syslogWriter.Close()
 			<-s.transportReconnect
+			break
 		}
 	}
 
@@ -426,9 +426,7 @@ PROCESS_LOOP:
 			// flush the buffer
 		case <-flushTimer.C:
 			if !s.syslogReady {
-				s.LogInfo("buffer cleared!")
 				bufferDm = nil
-				continue
 			}
 
 			if len(bufferDm) > 0 {

